src/components/brew-input.js

```diff
@@ -16,18 +16,18 @@ const FirstPanel = (props) => {
       <FancyDropdown label="Brew Method " name="brewMethod" handleChange={props.handleChange}
         options={testEquip}
         />
-      <FancyInput label="Brew Date" name="brewDate" handleChange={props.handleChange}
-        value={props.brewDate} />
-      <FancyButton name="dateButton"
-        handleSubmit={props.setDateToday}
-        //handleSubmit={()=>console.log("date")}
-        buttonText="Set Date to Today"/>
+      <FancyDayPickerInput
+          label="Brew Date"
+          name="brewDate"
+          placeholder={props.brewDate}
+          onDayChange={props.handleDayChange}
+        />
+      <FancyButton name="dateButton" onClick={props.setDateToday} buttonText="Set Date to Today"/>
```

Comment:
ok, looked into this further
1. the prop for the button is `handleSubmit` rather than `onClick`.
2. the `<FancyDayPickerInput />` needs to a `value={props.value}` prop so that you can call it with `<FancyDayPickerInput value={props.brewDate} />`